added more details about name badges (#5)
DutchPHPConference/conference-diversity-and-inclusion: Guideline.md

#### 9.1 Name badges
The purpose of a name badge is to identify people easily. As such, it should be easy to read! You should use a large, easy to read font. Sans serif fonts are generally regarded as a good choice. Make the name stand out and contrast well - avoid grey on grey! A font size of 18 points or bigger if you can make it fit would make a difference.

> Consider allowing people to define or even write in their prefered name, rather than simply printing their legal name and do not require photo ID but rather use a QR code send by e-mail to allow people to identify themselves. Furthermore allow space for people to define their preferred pronouns or [provide premade buttons/stickers](http://yupgup.com/pronouns.html).

Reviewer: In order to normalise the use of preferred pronouns, names and pronouns should be on the same item to prevent people who "disagree" with needing to specify pronouns from ignoring and not using such things when they are offered.

Author: thx! I just took your sentence in place of mine

> Also consider allowing people to define or even write in their prefered name, rather than simply printing their legal name and do not require photo ID but rather use a QR code send via email to allow people to identify themselves. Furthermore allow space for people to define their preferred pronouns or [provide premade buttons/stickers](http://yupgup.com/pronouns.html).

Reviewer 2: "send via email" -> "sent by e-mail"

> Consider allowing people to define or even write in their preferred name, rather than simply printing their legal name and do not require photo ID but rather use a QR code send by e-mail to allow people to identify themselves. In order to normalise the use of preferred pronouns, names and pronouns should be on the same item to prevent people who "disagree" with needing to specify pronouns from ignoring and not using such things when they are offered. Alternatively consider providing [provide premade buttons/stickers](http://yupgup.com/pronouns.html).

Reviewer: ```suggestion
Consider allowing people to define or even write in their preferred name, rather than simply printing their legal name and do not require photo ID but rather use a QR code send by e-mail to allow people to identify themselves. In order to normalize the use of preferred pronouns, names and pronouns should be on the same item to prevent people who "disagree" with needing to specify pronouns from ignoring and not using such things when they are offered. Alternatively consider providing [pre-made buttons/stickers](http://yupgup.com/pronouns.html) if this isn't possible.
```

This change suggestion is rather unwieldy due to the line length, so I've broken it up into easy-to-read diffs below :slightly_smiling_face: 

### Suggestion 1

Thanks for using my previous suggestion! Unfortunately, I write in British English while the rest of the document is written in American English.

```diff
-In order to normalise the use of preferred pronouns
+In order to normalize the use of preferred pronouns
```

### Suggestion 2

Updating the grammar around the link.

```diff
-Alternatively consider providing [provide premade buttons/stickers](http://yupgup.com/pronouns.html).
+Alternatively consider providing [pre-made buttons/stickers](http://yupgup.com/pronouns.html) if this isn't possible.
```

Author: thx again!

> Additionally it may make sense to provide stickers for people to denote things like "first time attendee", "ask me about [ - space to write in- ]", "talk to me" to forster more interaction between attendees.

Reviewer 2: This is the what but not the why. It would be nice if you could add why it may make sense. 

Also a typo, forster -> foster

Author: the document in general doesn't go into all that much detail about everything and instead provides links for further reading .. in this spirit imho "foster more interaction between attendees" is sufficient, but if you have a link it would be good to add it.

> Providing stickers or different lanyards (colors and patterns) to enable people to choose if they want photos taken of themselves and how these may be used. For inspiration see [here](https://twitter.com/imeluny/status/956072960296673280?lang=en).

Reviewer: Make sure that these identifying features are large enough to be seen post-event while processing photos for publication.

Author: thx again!

> Providing stickers or differently lanyards (colors and patterns) to enable people to choose if they want photos taken of themselves and how these may be used. For inspiration see [here](https://twitter.com/imeluny/status/956072960296673280?lang=en).

Reviewer 2: differently -> different

> Finally provide stickers or differently colored lanyards to enable people to choose if they want photos taken of themselves and how these may be used. For inspiration see [here](https://twitter.com/imeluny/status/956072960296673280?lang=en).

Reviewer 2: The word "Finally" in a living document can create bugs. If providing differently coloured lanyards, take colour blindness into account by also possibly having different patterns on them.
